cli.js

```diff
@@ -17,4 +17,14 @@ var cli = meow({
 	string: ['_']
 });
 
-cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite});
+try {
+	cpy([cli.input[0]], cli.input[1], {overwrite: cli.flags.overwrite}, function (err) {
```

Comment:
`cpy` should never throw so wrapping it in try shouldn't be necessary. If it does indeed throw we should catch it in `cpy` and pass the error in the callback.